### ISSUE TITLE
Glossary

### ISSUE THREAD

**Issue by telmogoncalves** [reactjs/pt-PT.reactjs.org#9]
## Termos a usar ao traduzir a documentação

| ✅ | ❌ |
| ------------- |:-------------:|
| hiperligação | link |
| passo a passo | passo-a-passo |
| node | nó |

**Comment by fjoshuajr** [comment]
E quanto ao uso de itálico para palavras estrangeiras. Como devemos usar?
Por exemplo: 
'o React vai preservá-lo e apenas atribuir manipuladores de eventos (_event handlers_)'

vs

'o React vai preservá-lo e apenas atribuir _event handlers_ (manipuladores de eventos)'

**Comment by imflavio** [comment]
@fjoshuajr Para mim faz mais sentido a segunda opçao.
Que traduçao usar para `rendering`?

**Comment by fjoshuajr** [comment]
> @fjoshuajr Para mim faz mais sentido a segunda opçao.
> Que traduçao usar para `rendering`?

@imflavio rendering -> renderizar

@telmogoncalves e a questão do node vs nó... acho que podemos fazer ' *node* (nó) '

**Comment by fjoshuajr** [comment]
@telmogoncalves e a questão do node vs nó... acho que podemos fazer '*node* (nó)'

**Comment by fjoshuajr** [comment]
Outra sugestão é a palavra wrapper:
you can use a wrapper -> podemos utilizar um _wrapper_
to wrap something -> encapsular algo

**Comment by imflavio** [comment]
@fjoshuajr Essas palavras nao existem no Portugues.. Na minha opiniao ou usamos um sinonimo (nao faço a minima ideia qual faria sentido aqui) ou a palavra em ingles. Renderizar nao me diz nada, nem nó no contexto da DOM: (https://developer.mozilla.org/pt-PT/docs/Web/API/Node)

**Comment by jbarradas** [comment]
Eu por acaso, no dia a dia, digo renderizar, mas sim a palavra não existe.
Talvez algo como mostrar/exibir/apresentar sirva no contexto? (fui buscar a 'to display')

**Comment by jbarradas** [comment]
Quanto à outra questão, prefiro a 1a opcao: "... manipuladores de eventos (eventos handlers)", pois estaríamos a continuar a frase em português, e dentro do parêntesis os helpers em inglês.

**Comment by imflavio** [comment]
Eu nao uso no dia a dia, mas se tivesse de usar acho que o mais natural seria "dar render". O pessoal do brasil usa renderizar, talvez nao seja muito mau.. "Pintar"?

**Comment by fjoshuajr** [comment]
> @fjoshuajr Essas palavras nao existem no Portugues.. Na minha opiniao ou usamos um sinonimo (nao faço a minima ideia qual faria sentido aqui) ou a palavra em ingles. Renderizar nao me diz nada, nem nó no contexto da DOM: ([developer.mozilla.org/pt-PT/docs/Web/API/Node](https://developer.mozilla.org/pt-PT/docs/Web/API/Node))

Ok, usemos node.
Já a palavra render acho que nao ha mal nenhum em usar renderizar, porque é daquelas palavras que apesar de nao ser originalmente portuguesa, tem sido usada cada vez mais. Tal como marketing.

**Comment by jbarradas** [comment]
Por um lado é verdade. No que toca a termos técnicos, nem sempre vamos ter palavras adequadas. E algumas delas acabam por ser inseridas no dicionário, tal como marketing.

Não me choca usarmos renderizar (desde que em itálico).

Quanto ao nó, acho preferível usarmos "nó (node)". Mas uma vez mais, não me choca se usarmos node em itálico, apenas acho que poderá soar um pouco estranho na frase.

**Comment by telmogoncalves** [comment]
Eu acho que palavras como `node` não deve sequer ter tradução. Visto ser uma _framework_ de JavaScript.

Em relação aos `event handlers`, eu prefiro como o @fjoshuajr disse:
```
manipuladores de eventos (_event handlers_)
```

**Comment by telmogoncalves** [comment]
> Por um lado é verdade. No que toca a termos técnicos, nem sempre vamos ter palavras adequadas. E algumas delas acabam por ser inseridas no dicionário, tal como marketing.
> 
> Não me choca usarmos renderizar (desde que em itálico).
> 
> Quanto ao nó, acho preferível usarmos "nó (node)". Mas uma vez mais, não me choca se usarmos node em itálico, apenas acho que poderá soar um pouco estranho na frase.

Podes usar `node`, em vez de ser itálico. Node é um runtime de JavaScript, tal como não vamos traduzir `React` para `Reagir`.

Como também vai traduzir `CoffeeScript` para `Roteiro de Café`.

**Comment by jbarradas** [comment]
(sorry estou na versão mobile e não consigo fazer quote)

Mas React e CoffeeScript aqui funcionam como nomes próprios. E não termos técnicos. Logo nunca se traduziriam.

Mas não achas um pouco estranho esta frase: 

"Se invocares ReactDOM.hydrate() em um node que já tem o seu markup processado pelo servidor" ?

**Comment by telmogoncalves** [comment]
> (sorry estou na versão mobile e não consigo fazer quote)
> 
> Mas React e CoffeeScript aqui funcionam como nomes próprios. E não termos técnicos. Logo nunca se traduziriam.
> 
> Mas não achas um pouco estranho esta frase:
> 
> "Se invocares ReactDOM.hydrate() em um node que já tem o seu markup processado pelo servidor" ?

Concordo, não soa muito bem, mas substitui `node` com `nó`, ainda é mais confuso. Acho que uma boa solução seria explicar logo no início o que é um `node`.

**Comment by jbarradas** [comment]
Hmm, sim, é sem dúvida uma opção!

Mas:
"Se invocares ReactDOM.hydrate() em um nó (node ) que já tem o seu markup processado pelo servidor"

Confesso que já não me parece tão estranho. Seja como for, acho que está visto que não vamos conseguir uma 'regra' e terá que ser visto caso a caso. 

Por mim sou a favor de "nó (node)", ou a explicação de node.

**Comment by telmogoncalves** [comment]
As palavras mais _complicadas_ acho que devemos usar sempre `coisas (_stuff_)`.

Usando o teu comentário: `[...] que já tem o seu markup (_código_)`.

Eu sinceramente nunca ouvi a expressão `nó` para `node` 😁Só preciso de saber mesmo qual a nossa conclusão para adicionar à lista.

Por mim pode ser `nó (node)` se acharem que faz mais sentido.

**Comment by fjoshuajr** [comment]
> > Por um lado é verdade. No que toca a termos técnicos, nem sempre vamos ter palavras adequadas. E algumas delas acabam por ser inseridas no dicionário, tal como marketing.
> > Não me choca usarmos renderizar (desde que em itálico).
> > Quanto ao nó, acho preferível usarmos "nó (node)". Mas uma vez mais, não me choca se usarmos node em itálico, apenas acho que poderá soar um pouco estranho na frase.
> 
> Podes usar `node`, em vez de ser itálico. Node é um runtime de JavaScript, tal como não vamos traduzir `React` para `Reagir`.
> 
> Como também ninguém vai traduzir `CoffeeScript` para `Roteiro de Café`.

Talvez eu tenha percebido mal aqui, mas node neste trecho 'Se invocares ReactDOM.hydrate() em um node que já tem o seu markup processado pelo servidor', refere-se ao nó de uma árvore -- neste caso a DOM. Ao passo que Node é o antigo NodeJS. 
Dei uma espreitadela no google e encontrei este [pdf](https://web.fe.up.pt/~rcamacho/cadeiras/bioinformatica/docs/arvores.pdf), nas páginas 2 e 3 falam de nós.

E já agora, é a DOM vs o DOM? Português é difícil 😜

**Comment by fjoshuajr** [comment]
> As palavras mais _complicadas_ acho que devemos usar sempre `coisas (_stuff_)`. 

👌

**Comment by telmogoncalves** [comment]
> > > Por um lado é verdade. No que toca a termos técnicos, nem sempre vamos ter palavras adequadas. E algumas delas acabam por ser inseridas no dicionário, tal como marketing.
> > > Não me choca usarmos renderizar (desde que em itálico).
> > > Quanto ao nó, acho preferível usarmos "nó (node)". Mas uma vez mais, não me choca se usarmos node em itálico, apenas acho que poderá soar um pouco estranho na frase.
> > 
> > 
> > Podes usar `node`, em vez de ser itálico. Node é um runtime de JavaScript, tal como não vamos traduzir `React` para `Reagir`.
> > Como também ninguém vai traduzir `CoffeeScript` para `Roteiro de Café`.
> 
> Talvez eu tenha percebido mal aqui, mas node neste trecho 'Se invocares ReactDOM.hydrate() em um node que já tem o seu markup processado pelo servidor', refere-se ao nó de uma árvore -- neste caso o DOM. Ao passo que Node é o antigo NodeJS.
> Dei uma espreitadela no google e encontrei este [pdf](https://web.fe.up.pt/~rcamacho/cadeiras/bioinformatica/docs/arvores.pdf), nas páginas 2 e 3 falam de nós.
> 
> E já agora, é a DOM vs o DOM?
> Espreitei no MDN, é o Modelo de Objecto de Documento (DOM)
> 
> Português é difícil 😜

Sim, vamos manter `nó (node)`, é capaz de ser mais fácil. As pessoas acabam por perceber do que realmente se trata o `node`. Eu já adicionei ao glosário.

**Comment by fjoshuajr** [comment]
Novos termos por adicionar:

| Inglês | ❌ | ✅ |
| ------------- |:-------------:|:-------------:|
| escape hatch | porta de salvação | escapatória |
| deprecated | deprecado | descontinuado |

Thoughts?

**Comment by jbarradas** [comment]
> Novos termos por adicionar:
> 
> Inglês		
> escape hatch	porta de salvação	escapatória
> deprecated	deprecado	descontinuado
> timestamp	carimbo temporal	data e hora
> Thoughts?

Estava à espera das reviews para adicionar.
Descontinuado, concordo.
Escapatória, percebo, mas no contexto em que usei, acho que vai soar estranho.
Carimbo temporal, parece-me super estranho.

No entanto, também não estou seguro com os termos que usei.

**Comment by luisdiogo2071317** [comment]
Eu precisava de saber como traduzir este pedaço no faq-function: React pools events

**Comment by antonio-rodrigues** [comment]
"React pooling event" significa algo como "Event sempre disponível".

> This is what we mean by 'pooling': the event object is in effect being sent back to the pool for use in a later event.

Faria sentido, em contexto, algo como "Eventos React sempre disponíveis"? Caso contrário diria apenas "Eventos React".

**Comment by antonio-rodrigues** [comment]
Concordo, "Eventos React (eventos da pool)" é explicito e em contexto.
